packages/shared/src/styles/devices.scss

```diff
@@ -60,3 +60,37 @@ $desktop-width: 1024px;
         @content;
     }
 }
+
```

Comment:
This is already added in the main branch . isnt?

Reply:
yes, not sure why it shows up in my PR